ci/build/upload_build_info.sh

```diff
@@ -31,6 +31,5 @@ mkdir -p "$BAZEL_LOG_DIR"
 
 ./ci/build/get_build_info.py > "$BAZEL_LOG_DIR"/metadata.json
 
-# Keep cryptography/openssl in sync with `requirements/test-requirements.txt`
-pip install -q -c "${RAY_DIR}/python/requirements.txt" docker aws_requests_auth boto3 cryptography==38.0.1 PyOpenSSL==23.0.0
+pip install -U -q -c "${RAY_DIR}/python/requirements.txt" docker aws_requests_auth boto3 urllib3 requests cryptography PyOpenSSL
```

Comment:
should you pin `requirements_compiled.txt` ?

Reply:
good catch